openstudio-workflow.gemspec

```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'ci_reporter_rspec', '~> 1.0.0'
   s.add_development_dependency 'coveralls', '~> 0.8.21'
   s.add_development_dependency 'json-schema', '~> 2.8.0'
+  s.add_development_dependency 'openstudio-standards', '~> 0.2.12'
```

Comment:
yuck, this is an unfortunate dependency to add.

Reply:
yep... bc of the openstudio results measure for testing

Reply:
blargh... okay